start.sh

```diff
@@ -11,6 +11,43 @@ python3 manage.py shell -c "from profiles.models import UserProfile; up=UserProf
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Rocket League')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Dota 2')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Hearthstone')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Fortnite')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='SMITE')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Rainbow Six Siege')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Heroes of the Storm')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Call of Duty: Modern Warfare')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Apex Legends')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Super Smash Bros. Ultimate')"
+python3 manage.py shell -c "from matches.models import GameChoice; GameChoice.objects.get_or_create(name='Clash Royale')"
+
+echo creating default sports choices
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Ice Hockey')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Soccer')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Indoor Soccer')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Futsal')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Roller Hockey')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Basketball')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Football')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Lacrosse')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Baseball')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Kan Jam')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Spikeball')"
+python3 manage.py shell -c "from matches.models import SportChoice; SportChoice.objects.get_or_create(name='Ultimate')"
+
```

Comment:
Rather than force create the defaults, we should have a way to choose (possibly with an environment variable)

Reply:
I personally don't see any need of having that. Create it, they can modify, delete or add to if they'd like. Just helps them get up and going faster. 
If its an esports site these SportChoice objects will be hidden and hardly ever accessed.

Reply:
The main issue is even if they delete the defaults they will be recreated every time the container restarts